modules/i18n/src/translations/en-US/portals/console.ts

```diff
@@ -26,66 +26,66 @@ export const console: ConsoleNS = {
                     gravatar: {
                         errors: {
                             noAssociation: {
-                                content: "It seems like the selected email is not registered on Gravatar. " +
-                                    "Sign up for a Gravatar account by visiting Gravatar official website or use " +
-                                    "one of the following.",
-                                header: "No matching Gravatar image found!"
+                                content: "Il semble que l'e-mail sélectionné ne soit pas enregistré sur Gravatar. " +
```

Comment:
This is the `en-US` translation file, hence french translations here is not needed right?

Reply:
good catch we have inverted fr and en file. I change it.